rs/registry/canister/src/mutations/node_management/do_add_node.rs

```diff
@@ -88,10 +89,17 @@ impl Registry {
             .domain
             .as_ref()
             .map(|domain| {
-                if !is_valid_domain(domain) {
-                    return Err(format!(
-                        "{LOG_PREFIX}do_add_node: Domain name `{domain}` has invalid format"
-                    ));
+                match domain_to_ascii_strict(domain) {
+                    Err(err) => return Err(format!(
+                        "{LOG_PREFIX}do_add_node: Domain name `{domain}` has invalid format: {err}"
```

Comment:
Test failure here is showing "err" is rendered as "Errors" - is there a way to expand that so it's useful to the caller?